### PR TITLE
Added Anne Ogborn as a mentor for Prolog

### PR DESCRIPTION
**Pull request by Anniepoo**
Add myself (as my AnnieAtDataChemist account) to the Prolog mentors

### CODE REVIEW

**Comment by Anniepoo** [comment]
ok, should be fixed now.

File: mentors/prolog/a.json [hunk 1]
```diff
@@ -1,10 +1,19 @@
 [
  {
```
Comment (rpottsoh) [inline mentors/prolog/a.json]: Line 2 should be indented a total of two spaces from the left.

File: mentors/prolog/a.json [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
-  "github_username": "Average-user",
-  "name": "Lucas Polymeris",
+  "github_username": "AnnieAtDataChemist",
+  "name": "Anne Ogborn",
   "link_text": null,
   "link_url": null,
   "avatar_url": null,
-  "bio": "I'd like to make Prolog a better known programming language."
- }
+  "bio": "Contributor to SWI-Prolog and all around Prolog enthusiast"
```
Comment (rpottsoh) [inline mentors/prolog/a.json]: Lines 3 through 8 should be indented a total of 4 spaces from the left.

File: mentors/prolog/a.json [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
+ },
```
Comment (rpottsoh) [inline mentors/prolog/a.json]:
```suggestion
  },
```
Line 9 should be indented a total 2 spaces, just like line 10.

File: mentors/prolog/a.json [hunk 4]
```diff
@@ -1,10 +1,19 @@
...
+  {
+  "github_username": "Average-user",
+  "name": "Lucas Polymeris",
+   "link_text": null,
+   "link_url": null,
+   "avatar_url": null,
+   "bio": "I'd like to make Prolog a better known programming language."
```
Comment (rpottsoh) [inline mentors/prolog/a.json]: Lines 11 through 16 should be indented a total (each) 4 spaces from the left.

**Comment by rpottsoh** [comment]
@Anniepoo otherwise your changes are fine.

**Comment by rpottsoh** [comment]
@Anniepoo the Travis CI configuration has been updated.  You need to update your fork and rebase our changes which will trigger Travis to run again using its updated configuration.  This is unrelated to the other changes I have requested.

**Comment by Anniepoo** [comment]
ok, this has become a circus. I'm dropping this PR and will submit a new one.

**Pull request closed without merging**